tools/question_converters/canvas/quiz2pl.py

```diff
@@ -222,12 +224,12 @@ def file_name_only(name):
                 blanks[answer['blank_id']].append(answer)
             question_text = question['question_text']
             for blank, answers in blanks.items():
-                dropdown = f'<pl-dropdown answers-name="{blank}">\n'
+                dropdown = f'<pl-multiple-choice display="dropdown" hide-letter-keys="true" answers-name="{blank}">\n'
```

Comment:
I no longer have easy access to a Canvas instance. Could someone (@firasm?) test that this is working as expected? In essence just convert a quiz that includes a question with "multiple dropdowns" type, and check that it is properly converted to PL as listed above.

Reply:
I can do this but it will take me some time because I'm currently buried in teaching. I know it'll only take 30-40 mins to try this.

I'll try and get a student who will be working on the script to try it.

Reply:
Just tested locally with a sandbox course I still have access to, and it works as expected. Thanks anyway @firasm.